### PR TITLE
Tweaks to various env and whitelabel global variables

### PR DESCRIPTION
**Pull request by bmedx**

### CODE REVIEW

**Comment by bmedx** [comment]
jenkins run e2e

**Comment by bmedx** [comment]
jenkins run e2e

File: regression/pages/whitelabel/__init__.py [hunk 1]
```diff
@@ -19,67 +19,119 @@
...
+            ECOM_PREFIX,
+            ECOM_BASE_URL
+        )
+    else:
+        LMS_URL_WITH_AUTH = '{}://{}'.format(LMS_PROTOCOL, BASE_URL)
+        ECOM_URL_WITH_AUTH = '{}://{}{}'.format(
+            LMS_PROTOCOL,
+            ECOM_PREFIX,
+            ECOM_BASE_URL
+        )
```
Comment (doctoryes) [inline regression/pages/whitelabel/__init__.py]: How about this instead?:
```
    BASIC_AUTH_URL = ''
    if BASIC_AUTH_USERNAME and BASIC_AUTH_PASSWORD:
        BASIC_AUTH_URL = '{}:{}@'.format(BASIC_AUTH_USERNAME, BASIC_AUTH_PASSWORD)

    LMS_URL_WITH_AUTH = '{}://{}{}'.format(
        LMS_PROTOCOL, BASIC_AUTH_URL, BASE_URL
    )
    ECOM_URL_WITH_AUTH = '{}://{}{}{}'.format(
        LMS_PROTOCOL,
        BASIC_AUTH_URL,
        ECOM_PREFIX,
        ECOM_BASE_URL
    )
```

Comment (doctoryes) [reply regression/pages/whitelabel/__init__.py]: Or maybe split this out into a private method since the code is cloned above?

**Comment by bmedx** [comment]
@doctoryes I've refactored the url string manipulation, care to take another look?

**Pull request merged**